scripts/Endergy.zs

```diff
@@ -0,0 +1,104 @@
+	rh(<enderio:item_alloy_endergy_ball:1>);
+	rh(<enderio:item_alloy_endergy_ball:4>);
+	rh(<enderio:item_alloy_endergy_ball:5>);
+	rh(<enderio:item_alloy_endergy_ball:6>);
+
+# Remove liquids of removed alloys
+	mods.tconstruct.Melting.removeRecipe(<liquid:crude_steel>);
+	mods.tconstruct.Melting.removeRecipe(<liquid:crystalline_alloy>);
+	mods.tconstruct.Melting.removeRecipe(<liquid:crystalline_pink_slime>);
+	mods.tconstruct.Melting.removeRecipe(<liquid:energetic_silver>);
+	mods.tconstruct.Melting.removeRecipe(<liquid:vivid_alloy>);
+
+	mods.nuclearcraft.melter.removeRecipeWithOutput(<liquid:crude_steel>);
+	mods.nuclearcraft.melter.removeRecipeWithOutput(<liquid:crystalline_alloy>);
+	mods.nuclearcraft.melter.removeRecipeWithOutput(<liquid:crystalline_pink_slime>);
+	mods.nuclearcraft.melter.removeRecipeWithOutput(<liquid:energetic_silver>);
+	mods.nuclearcraft.melter.removeRecipeWithOutput(<liquid:vivid_alloy>);
+
+# Crystalline Capacitor
+	recipes.remove(<enderio:item_capacitor_crystalline>);
+	recipes.addShapedMirrored("Crystaltine Capacitor",
+	<enderio:item_capacitor_crystalline>,
+	[[null,<ore:ingotCrystaltine>, null],
+	[<enderio:item_basic_capacitor:2>,<ore:dustPrismarine>,<enderio:item_basic_capacitor:2>],
+	[null,<ore:ingotCrystaltine>,null]]);
+
+# Anglesite - Crystal made of ThermalExpansion/EnvironmentalTech/Ender IO/AE2 materials
+	mods.extendedcrafting.CombinationCrafting.addRecipe(<contenttweaker:anglesite>, 100000000, 1000000, 
+	<appliedenergistics2:material:48>, [<thermalfoundation:material:894>,
+	<thermalfoundation:material:893>,  <thermalfoundation:material:895>, 
+	<thermalfoundation:material:865>,
+	<environmentaltech:litherite_crystal>, <environmentaltech:erodium_crystal>, 
+	<environmentaltech:kyronite_crystal>, <environmentaltech:pladium_crystal>, 
+	<environmentaltech:ionite_crystal>, <environmentaltech:aethium_crystal>,
+	<enderio:item_material:44>, <enderio:item_basic_capacitor:2>, 
+	<enderio:item_material:17>,  <enderio:item_material:18>, <enderio:item_material:19>]);	
+
+# Benitoite - Crystal made of Botania/AstralSorcery/BloodMagic/Thaumcraft materials
+#	recipes.remove(<biomesoplenty:crystal>);
+	mods.extendedcrafting.CombinationCrafting.addRecipe(<contenttweaker:benitoite>, 100000000, 1000000, <botania:manaresource:5>, 
+	[<botania:manaresource:9>, <botania:manaresource:1>, <botania:manaresource:7>, 
+	<botania:pylon:1>, <botania:manaresource:2>, <botania:manaresource:8>, 
+	<astralsorcery:itemcraftingcomponent:2>, #<astralsorcery:itemshiftingstar>.withTag({}), 
+	<astralsorcery:itemcraftingcomponent:4>, <astralsorcery:itemcoloredlens:6>, 
+	<bloodmagic:blood_rune:9>, <bloodmagic:blood_rune:10>, <bloodmagic:points_upgrade>,
+	<bloodmagic:slate:4>, #<astralsorcery:itemcelestialcrystal>.withTag({}),
+	#<thaumcraft:phial:1>.withTag({Aspects: [{amount: 10, key: "vitium"}]}), 
+	<thaumcraft:mechanism_complex>]);
+	
+	
+# Blutonium Block // Melodic Block
+//	recipes.remove(<minecraft:dirt>);
```

Comment:
errr? :P

Reply:
Yeah, still need tidbits of code cleanup :P